### ISSUE TITLE
Not OK: perl v5.7.0 +DEVEL8890 on powerpc-machten 4.1.4 (UNINSTALLED)

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#3481]
Migrated from [rt.perl.org#5908](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=5908) (status was 'resolved')

Searchable as RT5908$

**Comment by p5pRT** [comment]
### From domo@computer.org



base/commonsense\.\.\.\.\.FAILED at test 0
Failed a basic test\-\-cannot continue\.

The test \(and quite a few\,but by no means all\, others\) runs OK when
invoked separately and\, as you can see\, miniperl's healthy enough to
create this report\. The trouble seems to be a perlio stdout buffering
problem I failed to fix some while ago \-\- see bug report 20001228\.001\.
I'm terribly busy on other stuff just now\, but I'll have another shot
at chasing it down\.

The good news&#8203;: MachTen got patched to work with Mac OS 9\.1\, so life's
better in that respect\.

<details><summary>Perl Info</summary>

```

Flags:
     category=install
     severity=none

Site configuration information for perl v5.7.0:

Configured by domo at Sun Feb 25 17:40:01 WET 2001.

Summary of my perl5 (revision 5.0 version 7 subversion 0) configuration:
   Platform:
     osname=machten, osvers=4.1.4, archname=powerpc-machten
     uname='machten computer 5 0.5 powerpc '
     config_args='-de -Dusedevel'
     hint=recommended, useposix=true, d_sigaction=define
     usethreads=undef use5005threads=undef useithreads=undef 
usemultiplicity=undef
     useperlio=define d_sfio=undef uselargefiles=define usesocks=undef
     use64bitint=undef use64bitall=undef uselongdouble=undef
   Compiler:
     cc='cc', ccflags ='-DNOTDEF_MACHTEN -DREG_INFTY=2047 -I/usr/local/include',
     optimize='-O2 -fomit-frame-pointer',
     cppflags='-DNOTDEF_MACHTEN -DREG_INFTY=2047 -I/usr/local/include'
     ccversion='', gccversion='2.8.1', gccosandvers='machten4'
     intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=4321
     d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=8
     ivtype='long', ivsize=4, nvtype='double', nvsize=8, 
Off_t='off_t', lseeksize=4
     alignbytes=8, usemymalloc=y, prototype=define
   Linker and Libraries:
     ld='ld', ldflags =' -Xlstack=1048576 -L/usr/local/lib'
     libpth=/usr/local/lib /usr/lib
     libs=-lndbm -lgdbm -ldb -lm -lc -lutil
     perllibs=-lndbm -lgdbm -ldb -lm -lc -lutil
     libc=/usr/lib/libc.a, so=so, useshrplib=false, libperl=libperl.a
   Dynamic Linking:
     dlsrc=dl_none.xs, dlext=none, d_dlsymun=undef, ccdlflags=''
     cccdlflags='', lddlflags=''

Locally applied patches:
     DEVEL8890


@INC for perl v5.7.0:
     lib
     /usr/local/lib/perl5/5.7.0/powerpc-machten
     /usr/local/lib/perl5/5.7.0
     /usr/local/lib/perl5/site_perl/5.7.0/powerpc-machten
     /usr/local/lib/perl5/site_perl/5.7.0
     /usr/local/lib/perl5/site_perl/5.6.0/powerpc-machten
     /usr/local/lib/perl5/site_perl/5.6.0
     /usr/local/lib/perl5/site_perl
     .


Environment for perl v5.7.0:
     HOME=/home/domo
     LANG (unset)
     LANGUAGE (unset)
     LD_LIBRARY_PATH=/usr/lib
     LOGDIR (unset)
 
PATH=/sbin:/usr/sbin:/home/domo/bin:/usr/local/bin:/usr/bin:/bin:/usr/X11/bin:/usr/libexec
     PERL_BADLANG (unset)
     SHELL=/bin/sh

Get Your Private, Free Internet Access from FREE INTERNET at http://www.free-internet.lu

```
</details>

**Comment by p5pRT** [comment]
@jhi - Status changed from 'open' to 'resolved'
